ocaml/xapi/dbsync_master.ml

```diff
@@ -287,6 +289,41 @@ let ensure_vm_metrics_records_exist_noexn __context =
   Helpers.log_exn_continue "ensuring VM_metrics flags exist"
     ensure_vm_metrics_records_exist __context
 
+let setup_telemetry ~__context =
+  let pool = Helpers.get_pool ~__context in
+  let ref = Db.Pool.get_telemetry_uuid ~__context ~self:pool in
+  if ref = Ref.null then
+    Helpers.log_exn_continue "Setting up telemetry"
+      (fun () ->
+        Helpers.call_api_functions ~__context (fun rpc session_id ->
+            Client.Pool.reset_telemetry_uuid ~rpc ~session_id ~self:pool
+        ) ;
+        (* An exception will result in leaving the next schedule as default *)
+        let bound_hour =
```

Comment:
The name `bound` doesn't speak to me. This looks to me like `interval_hours`

Reply:
This is the `bound` of [Random.int](https://v2.ocaml.org/api/Random.html).
I added a comment for clarification.